tests/scraper/crawler/crawler_test.py

```diff
@@ -0,0 +1,127 @@
+from c4v.scraper.crawler.crawlers.el_pitazo_crawler import ElPitazoCrawler
+from c4v.scraper.crawler.crawlers.primicia_crawler import PrimiciaCrawler
+
+
+def test_crawler_parse_el_pitazo_index():
+    elp = ElPitazoCrawler()
+
+    out = elp.parse_sitemaps_urls_from_index(el_pitazo_sitemap_index())
+    ans = ["https://elpitazo.net/sitemap-pt-post-2021-05.xml", "https://elpitazo.net/sitemap-pt-post-2021-04.xml", "https://elpitazo.net/sitemap-pt-post-2021-03.xml"]
+
+    assert set(out) == set(ans)
+
+def test_crawler_parse_el_pitazo_sitemap():
+    elp = ElPitazoCrawler()
+
+    out = elp.parse_urls_from_sitemap(el_pitazo_sitemap_body())
+    ans = ["https://elpitazo.net/gran-caracas/el-pitazo-sigue-en-pie-para-vencer-la-censura-y-el-bloqueo/", "https://elpitazo.net/cultura/ozuna-anuncia-concierto-virtual-gratuito-en-youtube/", "https://elpitazo.net/politica/almagro-insiste-en-seguir-denunciando-crimenes-de-lesa-humanidad/"]
+
+    assert set(out) == set(ans)
+
+def test_crawler_parse_primicia_index():
```

Comment:
would suggest making two separate test files
one for el pitazo crawler and one for el primicia crawler - they are two separate classes so should have tests in different files I think